Gaijinizer patch/Map038.txt

```diff
@@ -4,108 +4,110 @@
 
 なんとかシェルターを抜けれたな。
 > CONTEXT: Map038/events/4/pages/0/5/Dialogue/yuuma
-I thought what would happen at one time ...
-I managed to get out of the shelter.#MTLed
+For a while there, I wasn't sure how things were
+going to turn out...
+But you've managed to get out of there, huh.#Ron
 > END STRING
 
 > BEGIN STRING
 地上に出たら、市街地を抜けて、俺のいる港を目指す。
 
 その港に着いたら、一緒にこの地域を脱出しよう。
 > CONTEXT: Map038/events/4/pages/0/9/Dialogue/yuuma
-When I get out on the ground, I will go through
-the city and aim for the port where I am.
-When you arrive at the port, let's escape this
-area together.#MTLed
+Once you're above ground, head through the city
+and try to get to the port where I am.
+We'll escape together when you get to the port.#Ron
 > END STRING
 
 > BEGIN STRING
 なんだ………？
 
 なにか落ちてる。
 > CONTEXT: Map038/events/4/pages/0/17/Dialogue/kazuya
-what………? Something is falling.#MTLed
+Huh...?
+
+There's something on the floor.#Ron
 > END STRING
 
 > BEGIN STRING
 テープレコーダー…………？
 > CONTEXT: Map038/events/4/pages/0/27/Dialogue/kazuya
-Tape recorder ............?#MTLed
+A tape recorder...?#Ron
 > END STRING
 
 > BEGIN STRING
 ………！！！バカな！なぜそれがそこに！？
 
 俺が市街地で廃棄したはず……！！
 > CONTEXT: Map038/events/4/pages/0/29/Dialogue/yuuma
-.........!!! stupid! Why is it there! ?
-I should have disposed of it in the city area
-...! !#MTLed
+...!!! No way! How did that get there!?
+I'm sure I got rid of every last one while I was
+in the city!!#Ron
 > END STRING
 
 > BEGIN STRING
 ……………どういうことだよ、ユウマ。
 > CONTEXT: Map038/events/4/pages/0/34/Dialogue/kazuya
-......... What does that mean, Yuuma.#MTLed
+... What's this all about, Yuuma?#Ron
 > END STRING
 
 > BEGIN STRING
 僕は、このテープレコーダーをよく知ってる。
 
 僕がレイナにプレゼントした限定品だ。
 > CONTEXT: Map038/events/4/pages/0/38/Dialogue/kazuya
-I know this Tape recorder well.
-It is a limited item that I gave to Reina.#MTLed
+I know this tape recorder.
+It's a limited edition model I'd given to Reina.#Ron
 > END STRING
 
 > BEGIN STRING
 『このご時世、こんな古臭いものプレゼントするのは
 
 　お前くらいだ』って、四人で笑いあったよな。
 > CONTEXT: Map038/events/4/pages/0/42/Dialogue/kazuya
-"This time of age, to give such an old-fashioned
-present. It's about you, you four laughed.#MTLed
+She said, "You're giving me this old-fashioned
+crap in this day and age? That's just like you."
+We all laughed.#Ron
 > END STRING
```

Comment:
"Crap" would be something that Kazuya would say, but I don't think Reina would phrase it like that. There are some reasons for that you probably wouldn't have seen yourself since I don't think you've fully completed the game yet.

I'd rephrase it to something like "antique".

>_**She had said, "You're giving me this old-fashioned
antique in this day and age? That's just like you."
We had all laughed about that.**_